File: java/8.9/example8.java
import java.util.Scanner;

public class example8 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter a message");
        String stringInput = scanner.nextLine();
        String newString = stringInput.substring(stringInput.length() - 3, stringInput.length());
        System.out.println(newString);
    }
}

File: rust/src/e89/example8.rs
use substring::Substring;

pub fn main() {
    let mut stringinput = String::new();
    println!("Enter a string");
    std::io::stdin().read_line(&mut stringinput).unwrap();
    let newstring = stringinput.trim().substring(stringinput.trim().len()-3, stringinput.trim().len());
    println!("{}", newstring);
}